app/Http/Controllers/Order/ExtendedBaseInvoiceController.php

```diff
@@ -157,7 +157,9 @@ public function multiplePayment($clientid,$invoiceChecked, $payment_method,
                     if ($invoice) {
                         if ($total_paid >= $invoice->grand_total) {
                             $invoice_status = 'success';
-                        }
+                        } elseif($total_paid > 0) {
```

Comment:
which of the above `if` condition will be true when $total_paid is 0? In that case variable `$invoice_status` will be undefined and will throw exception

Reply:
It will not be 0. Added validation for the same. 
Reason: When admin goes for creating a payment manually, he cannot create a payment with 0 value